stripe/http_client.py

```diff
@@ -77,6 +79,10 @@ def new_default_http_client(*args, **kwargs):
 
 
 class HTTPClient(object):
+    MAX_RETRIES = 3
+    MAX_DELAY = 2
+    INITIAL_DELAY = 0.5
```

Comment:
Not so sure what's the best way to manage these constants here

Reply:
I think we should be consistent with stripe-ruby: the maximum number of retries should be configurable (and default to 0), so it should be a top-level variable in the `stripe` module.

The other two parameters are read-only in stripe-ruby, so I think it's fine to keep them as class constants here.